crates/data-model/src/upstream_oauth2/session.rs

```diff
@@ -19,12 +19,14 @@ pub enum UpstreamOAuthAuthorizationSessionState {
         completed_at: DateTime<Utc>,
         link_id: Ulid,
         id_token: Option<String>,
+        userinfo: Option<String>,
```

Comment:
I'm fine with adding a dependency to `serde_json` here, make that a `serde_json::Value`